diff --git a/rqalpha/mod/rqalpha_mod_sys_analyser/mod.py b/rqalpha/mod/rqalpha_mod_sys_analyser/mod.py
index 44430f28..dcc5a024 100644
--- a/rqalpha/mod/rqalpha_mod_sys_analyser/mod.py
+++ b/rqalpha/mod/rqalpha_mod_sys_analyser/mod.py
@@ -285,6 +285,7 @@ class AnalyserMod(AbstractMod):
 
         summary = {
             'strategy_name': strategy_name,
+            'benchmark': self._env.config.base.benchmark,
             'start_date': self._env.config.base.start_date.strftime('%Y-%m-%d'),
             'end_date': self._env.config.base.end_date.strftime('%Y-%m-%d'),
             'strategy_file': self._env.config.base.strategy_file,
@@ -301,6 +302,7 @@ class AnalyserMod(AbstractMod):
             'alpha': self._safe_convert(risk.alpha, 3),
             'beta': self._safe_convert(risk.beta, 3),
             'sharpe': self._safe_convert(risk.sharpe, 3),
+            'benchmark_sharpe': self._safe_convert(risk.benchmark_sharpe, 3),
             'excess_sharpe': self._safe_convert(risk.excess_sharpe, 3),
             'information_ratio': self._safe_convert(risk.information_ratio, 3),
             'downside_risk': self._safe_convert(risk.annual_downside_risk, 3),
@@ -310,6 +312,7 @@ class AnalyserMod(AbstractMod):
             'excess_volatility': self._safe_convert(risk.excess_volatility, 3),
             'excess_annual_volatility': self._safe_convert(risk.excess_annual_volatility, 3),
             'max_drawdown': self._safe_convert(risk.max_drawdown, 3),
+            'benchmark_max_drawdown': self._safe_convert(risk.max_drawdown, 3),
             'excess_max_drawdown': self._safe_convert(risk.excess_max_drawdown),
             'excess_returns': self._safe_convert(risk.excess_return_rate, 6),
             'excess_annual_returns': self._safe_convert(risk.excess_annual_return, 6)
